app/assets/javascripts/controllers/container_topology/container_topology_controller.js

```diff
@@ -6,15 +6,13 @@ angular.module('topologyApp', ['kubernetesUI', 'ui.bootstrap'])
   $scope.vs = null;
 
   $scope.refresh = function() {
-    var id;
-    if ($location.absUrl().match("show/$") || $location.absUrl().match("show$")) {
-      id = '';
-    } else {
-      id = '/'+ (/container_topology\/show\/(\d+)/.exec($location.absUrl())[1]);
+    result = (/\/([^\/]*)\/show(\/(\d+))?/.exec($location.absUrl()));
```

Comment:
> please explain what is the following change doing (lines 9-12)

for example:
gets ```entity_name``` and ```14``` out of ```server.com/entity_name/show/14```
This way, this controller can be used by any entity. before it would search for ```container_topology``` as an entity name specifically